docs/user/README.md

```diff
@@ -88,11 +93,6 @@ context name "1" and delete that cluster.
 Recall that cluster context names are prefixed with `kind-` so the default
 cluster name is `kind-1`.
 
-If you want the `create cluster` command to block until the control plane
-reaches a ready status, you can use the `--wait` flag and specify a timeout.
-To use `--wait` you must specify the units of the time to wait. For example, to
-wait for 30 seconds, do `--wait 30s`, for 5 minutes do `--wait 5m`, etc.
-
```

Comment:
Thank you for you PR @rimusz but could you explain your reasoning?
The `--wait` flag enables a timeout for any control plane nodes to each a `Ready` status at start up time (`kind create cluster`).
Why do yout hink this would make more sense in the "deleting a cluster" section?

Reply:
Haha I got the red and greed backwards! Thank you for catching this!